javaslang/src/main/java/javaslang/Value.java

```diff
@@ -1225,11 +1234,8 @@ default CharSeq toCharSeq() {
 }
 
 interface ValueModule {
-
-    @SuppressWarnings("unchecked")
-    static <T extends Traversable<V>, V> T toTraversable(Value<V> value, T empty,
-            Function<V, T> ofElement,
-            Function<Iterable<V>, T> ofAll) {
+    static <T extends Traversable<V>, V>
+    T toTraversable(Value<V> value, T empty, Function<V, T> ofElement, Function<Iterable<V>, T> ofAll) {
```

Comment:
they had 3 different styles, I've unified them.
Would you like me to revert or to convert all to the same style?